<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeySignatureManagerForm</class>
 <widget class="QWidget" name="KeySignatureManagerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Key Signature Manager</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QGroupBox" name="editorGroupBox">
     <property name="title">
      <string>Key signature editor</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="keyLabel">
          <property name="text">
           <string>Key</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="keyComboBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="orderLabel">
          <property name="text">
           <string>Order</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="stepLabel">
          <property name="text">
           <string>Step</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="orderSpinBox"/>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="stepSpinBox"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="createPushButton">
        <property name="text">
         <string>Create New</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="updatePushButton">
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="removePushButton">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearAllPushButton">
       <property name="text">
        <string>Clear All</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" rowspan="3">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="1" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listWidget</tabstop>
  <tabstop>removePushButton</tabstop>
  <tabstop>clearAllPushButton</tabstop>
  <tabstop>keyComboBox</tabstop>
  <tabstop>orderSpinBox</tabstop>
  <tabstop>stepSpinBox</tabstop>
  <tabstop>createPushButton</tabstop>
  <tabstop>updatePushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
